.coderabbit.yaml

```diff
@@ -26,10 +26,14 @@ reviews:
         - Proper use of rst syntax (e.g., avoid broken links or invalid code blocks).
         - Adherence to style guides (e.g., consistent tone, terminology).
         Suggest improvements to enhance readability and accessibility.
+        - Avoid suggesting new changes after applying fixes requested by you.
+        Suggest improvements on the first review only.
```

Comment:
Can this be worded to something like "Make sure to find all improvements on the first pass, go through the material multiple times before submitting comments."